src/main/java/com/tradeshift/amqp/autoconfigure/TunedRabbitAutoConfiguration.java

```diff
@@ -154,7 +155,7 @@ public EnableRabbitRetryAndDlqAspect enableRabbitRetryAndDlqAspect(TunedRabbitPr
             matchIfMissing = true)
     @Bean
```

Comment:
Since this guy is the first one in the flow, we need to wait the creation of the `rabbitComponentsFactory`.
```suggestion
    @Bean
    @DependsOn("rabbitComponentsFactory")
```